src/utilities/solve_eq_sys.jl

```diff
@@ -0,0 +1,37 @@
+"""
+Solves the equation system Ka = f taking into account the Dirichlet boundary conditions in the matrix bc
+"""
+function solve_eq_sys(K::AbstractMatrix, f::Array, bc::Matrix )
+
+    (nr, nc) = size(K);
+    if nr != nc
+        DimensionMismatch(@sprintf("Stiffness matrix is not square (#rows=%d #cols=%d)", nr, nc))
```

Comment:
I en sträng i Julia som innehåller `$` så evalueras det som står efter och sätts in i strängen.

T.ex:
```julia
julia> print("$(1+2)")
3
```

```julia
julia> a = 5
5

julia> print("a = $a")
a = 5
```

(Detta kallas string interpolation https://en.wikipedia.org/wiki/String_interpolation)

Du skulle alltså kunna skriva det lite enklare som:
```julia
DimensionMismatch("Stiffness matrix is not square (#rows=$nr, #cols=$nc)")
```

Du behöver också lägga till en `throw` så det blir:
```julia
throw(DimensionMismatch("Stiffness matrix is not square (#rows=$nr, #cols=$nc)"))
```